src/Kernel-Tests/MagnitudeTest.class.st

```diff
@@ -0,0 +1,8 @@
+"
+A MagnitudeTest is a test class for testing the behavior of Magnitude
+"
+Class {
+	#name : #MagnitudeTest,
```

Comment:
I see a new class without any new method... YAGNI or WIP?

Reply:
You seem to be better with abbreviations than I do :-).
However, you are right, and I opened issue #4730 to that matter, and hope to have a fix sometime today.